tiny_dnn/core/backend_tiny.h

```diff
@@ -42,93 +45,87 @@ namespace tiny_dnn {
 namespace core {
 
 class tiny_backend : public backend {
- public:
+   public:
```

Comment:
@beru Repeating same comment on same automatic change makes the PR very messy. Could you please leave one comment per one problem?